release/ray_release/cluster_manager/cluster_manager.py

```diff
@@ -74,3 +74,6 @@ def get_cluster_url(self) -> Optional[str]:
         if not self.project_id or not self.cluster_id:
             return None
         return anyscale_cluster_url(self.project_id, self.cluster_id)
+
+    def get_session_name(self) -> str:
```

Comment:
So cluster address returns 

proj/session_name

And it seems like this is not working for the prod, and we should just pass

session_name directly. Do you see any potential issue if we change cluster address to only return the cluster name?

Reply:
The reason why we're using proj/session name is because prod will require this in the future (in fact I got several deprecation warnings for that, which is why I made the change). The current job file manager also uses the full cluster address (but works with the CLI), so I think it should work?

If it doesn't work that's a bug and we should file it with prod. In that case we can work around this for now, but let's use `get_session_name(full=False)` or something so we don't duplicate this logic here (`full=True` should be the default though).